packages/eslint-plugin/test/__file_snapshots__/types/expect-diagnostics/index.d.ts.lint

```diff
@@ -0,0 +1,8 @@
+No errors
+
+==== types/expect-diagnostics/index.d.ts ====
+
+    export const foo = 1234;
+    
+    // This should cause a diagnostic.
```

Comment:
no diagnostic here

Reply:
Renamed and changed the comment, thanks for noticing.